tools/mo/openvino/tools/mo/convert_impl.py

```diff
@@ -216,6 +217,16 @@ def raise_ie_not_found():
         log.error(e)
         raise_ie_not_found()
 
+    if 'compress_to_fp16' in argv and argv.compress_to_fp16 and \
+            'data_type' in argv and argv.data_type in ['FP32', 'float']:
```

Comment:
```suggestion
            'data_type' in argv and argv.data_type:
```

Reply:
If we do that it will always fail, because even if user didn't specify any value at all cli_parser will se default values. Therefore need to fail only if `compress_to_fp16` and `data_type`values are conflicting.

If `---compress_to_fp16`/`--compress_to_fp16=True` and `--data_type=FP16` that's not a failure.